### DISCUSSION TITLE
Error: unknown command "d" for "yq"

### DISCUSSION THREAD

**Discussion by gberlotperalta** [mikefarah/yq#1058] [General]
I'm trying to run yq d command an the command isn't recognized
Even the d command isn't listed in the list of available commands, which version support that command.

Error: unknown command "d" for "yq"

https://github.com/mikefarah/yq/releases/tag/v4.16.2

**Comment by mikefarah** [comment]
The 'd' command is from version 3 of `yq` (which is now deprecated). To delete use this newer syntax: https://mikefarah.gitbook.io/yq/operators/delete

**Reply by mikefarah** [reply]
Also, if you were using v3 before, you may find this upgrade guide handy: https://mikefarah.gitbook.io/yq/upgrading-from-v3
